tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html

```diff
@@ -4155,6 +4165,32 @@ <h2>Show similarity to selected datapoint</h2>
           );
         },
 
+        /**
+         * Get the key to which this example belongs when the dataset is sliced by the given feature(s).
+         */
+        getSliceKey_: function(example, feature1, feature2) {
+          const bucketEdges = this.featureBucketEdges_;
+          function maybeAggregate(feature) {
+            const edges = bucketEdges[feature];
+            if (edges) {
+              for (let i = 1; i < edges.length; i++) {
+                if (
+                  example[feature] < edges[i] ||
```

Comment:
how does this handle when an example is missing the feature value for the selected feature?